app/src/ui/preferences/appearance.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+//import { Row } from '../lib/row'
+import { DialogContent } from '../dialog'
+import {
+  VerticalSegmentedControl,
+  ISegmentedItem,
+} from '../lib/vertical-segmented-control'
+import { ApplicationTheme } from '../lib/application-theme'
+import { fatalError } from '../../lib/fatal-error'
+
+interface IAppearanceProps {
+  readonly selectedTheme: ApplicationTheme
+  readonly onSelectedThemeChanged: (theme: ApplicationTheme) => void
+}
+
+const themes: ReadonlyArray<ISegmentedItem> = [
+  { title: 'Light', description: 'The default theme of GitHub Desktop' },
+  {
+    title: 'Dark (beta)',
+    description:
+      'A beta version of our coming dark theme. Still under development. Please report any issues you may find to our issue tracker.',
```

Comment:
the phrase `coming dark theme` sounds weird to me. thoughts @donokuda? i could also just see us remove the word `coming`

Reply:
> i could also just see us remove the word coming

I think this is what we should do. Language fail on my part